slasher/detection/service.go

```diff
@@ -161,3 +161,17 @@ func (ds *Service) submitAttesterSlashings(ctx context.Context, slashings []*eth
 		}
 	}
 }
+
+func (ds *Service) submitProposerSlashing(ctx context.Context, slash *ethpb.ProposerSlashing) {
+	ctx, span := trace.StartSpan(ctx, "detection.submitProposerSlashing")
+	defer span.End()
+	if slash != nil && slash.Header_1 != nil && slash.Header_2 != nil {
+		log.WithFields(logrus.Fields{
+			"header1Slot":         slash.Header_1.Header.Slot,
+			"header2Slot":         slash.Header_2.Header.Slot,
+			"proposerIdxHeader1": slash.Header_1.Header.ProposerIndex,
+			"proposerIdxHeader2": slash.Header_2.Header.ProposerIndex,
+		}).Info("Found an proposer slashing! Submitting to beacon node")
```

Comment:
The slot and proposer index are always gonna be the same if it is a slashing right? Maybe we can log out the truncated block roots instead of the same info twice?